src/app/clusters/thread-border-router-management-server/thread-border-router-management-server.h

```diff
@@ -76,14 +76,15 @@ class ServerInstance : public CommandHandlerInterface,
     {
         return HandleGetDatasetRequest(isOverCASESession, Delegate::DatasetType::kPending, dataset);
     }
-    Status HandleSetActiveDatasetRequest(CommandHandler * commandHandler,
+    Status HandleSetActiveDatasetRequest(bool isOverCASESession, CommandHandler * commandHandler,
                                          const Commands::SetActiveDatasetRequest::DecodableType & req);
-    Status HandleSetPendingDatasetRequest(const Commands::SetPendingDatasetRequest::DecodableType & req);
+    Status HandleSetPendingDatasetRequest(bool isOverCASESession, const Commands::SetPendingDatasetRequest::DecodableType & req);
     Status HandleGetDatasetRequest(bool isOverCASESession, Delegate::DatasetType type, Thread::OperationalDataset & dataset);
 
     // Attribute Read handlers
     void ReadFeatureMap(BitFlags<Feature> & feature);
     Optional<uint64_t> ReadActiveDatasetTimestamp();
+    Optional<uint64_t> ReadPendingDatasetTimestamp();
```

Comment:
New code like this cluster should be using `std::optional` directly instead of `Chip::Optional`

Reply:
Done.